utils/utils/src/main/scala/pl/touk/nussknacker/engine/util/TimestampUtils.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.util
+
+import java.sql.Timestamp
+import java.time.{Instant, LocalDate, OffsetDateTime, ZonedDateTime}
+import scala.util.Try
+
+object TimestampUtils {
+  // some timestamps (direct input from user or in data sources) can be in seconds
+  def normalizeTimestampToMillis(l: Long): Long =
+    if (l > 31536000000L) { // 1971.01.01 00:00:00 timestamp in milliseconds
+      l
+    } else { // l is nearly certainly given in seconds
+      l * 1000
+    }
+
+  def supportedTypeToMillis(value: Any, errorMsg: String): Long = value match {
+    case v: Long => normalizeTimestampToMillis(v)
+    case v: Int => normalizeTimestampToMillis(v)
+    case v: Instant => v.toEpochMilli
+    case v: LocalDate => Timestamp.valueOf(v.atStartOfDay()).getTime
+    case v: ZonedDateTime => Timestamp.valueOf(v.toLocalDateTime).getTime
```

Comment:
Please no. It is not the correct way to handle it. What about dates that are duplicated during a year because of DST? Please use `.toInstant`. `LocalDateTime` is in most cases bad choice for programmable usage. You should very rarely use it.

Reply:
is    
 `case v: LocalDate => v.atStartOfDay(ZoneId.systemDefault()).toInstant.toEpochMilli`
ok?

Reply:
This case led us to the place that LocalDate is tricky. Lets imagine situation when you have scenario with join with LocalDate on one branch and Instant on second. He think about LocalDate in Autralia because he know that these events are from Australia. How you will know this time zone?

Reply:
it's not possible to sensibly handle without additional info about time zone, and for example, avro docs specifically mention that `date` doesn't carry information about time-zone

I think it's more trouble than it's worth, I doubt timestamp in date (without time) format are used with any kind of popularity, i'll just remove support for LocalDate